### DISCUSSION TITLE
Funzionalità amministatore delle segnalazioni

### DISCUSSION THREAD

**Discussion by mbolzan** [globaleaks/globaleaks-whistleblowing-software#3687] [Q&A] [answered]
Sarebbe possibile configurare un ruolo (o abilitare quello di Amministratore) ad essere il "distributore" delle segnalazioni ai riceventi/canali? Ovvero una figura che valuti inizialmente la segnalazione e la assegni ad uno dei riceventi (o un gruppo di riceventi) affinché solo così essi la vedano e la elaborino?

thanks

**Comment by evilaliv3** [comment, marked as answer]
Si è possibile, bisogna:
- associare al canale questo solo utente
- assegnare a questo utente il privilegio di concedere visibilità delle segnalazioni ad altri utenti
